cot-macros/src/model.rs

```diff
@@ -91,10 +93,20 @@ impl ToTokens for ModelBuilder {
 impl ModelBuilder {
     fn from_model(model: Model) -> Self {
         let field_count = model.field_count();
+        let app_name = std::env::var("CARGO_PKG_NAME").unwrap();
```

Comment:
Please don't use `unwrap()` unless it's test code or something similar.
```suggestion
        let app_name = std::env::var("CARGO_PKG_NAME").expect("cargo should set the `CARGO_PKG_NAME` environment variable");
```

Reply:
Done.